RobotCode/src/main/java/frc/robot/constants/Ports.java

```diff
@@ -3,35 +3,29 @@
 public final class Ports
 {
 
-	// place holder values for intake motors
-	public static final int INTAKE_ON_MOTOR = 3;
-	public static final int INTAKE_MOTOR_RIGHT = 4;
-	public static final int INTAKE_MOTOR_LEFT = 5;
-
-	public final static int PNEUMATICS_HUB = 42;
+	// Intake ports
+	public static final int INTAKE_ROLLER_MOTOR_TOP = 0;
+	public static final int INTAKE_ROLLER_MOTOR_BOTTOM = 0;
+	public static final int INTAKE_DEPLOY_MOTOR_RIGHT = 0;
+	public static final int INTAKE_DEPLOY_MOTOR_LEFT = 0;
 
+	// Shooter mount ports
 	public static final int SHOOTER_MOUNT_MOTOR_LEFT = 0;
-	public static final int SHOOTER_MOUNT_MOTOR_RIGHT = 1;
+	public static final int SHOOTER_MOUNT_MOTOR_RIGHT = 0;
 
-	// Provvisory Climber Ports
-	public static final int CLIMBER_LEFT_CENTER_MOTOR = 7;
-	public static final int CLIMBER_RIGHT_CENTER_MOTOR = 8;
+	// Climber ports
+	public static final int CLIMBER_LEFT_MOTOR = 0;
+	public static final int CLIMBER_RIGHT_MOTOR = 0;
 
-	public static final int CLIMBER_OUTER_LEFT_MOTOR = 15;
-	public static final int CLIMBER_OUTER_RIGHT_MOTOR = 16;
-	public static final int CLIMBER_RETRACT_LEFT_MOTOR = 17;
-	public static final int CLIMBER_RETRACT_RIGHT_MOTOR = 18;
-	public static final int CLIMBER_EXTEND_LEFT_MOTOR = 17;
-	public static final int CLIMBER_EXTEND_RIGHT_MOTOR = 18;
-
-	// SHOOTER PORT WOOOOOO
+	// Shooter ports
 	public static final int SHOOTER_MOTOR_MAIN = 0;
 	public static final int SHOOTER_MOTOR_SUB = 0;
 	public static final int INDEXER_MOTOR_MAIN = 0;
 	public static final int INDEXER_MOTOR_SUB = 0;
 
-	// sensors and stuff
+	// Sensors and stuff
 	public static final int PIGEON_GYRO = 0;
+	public final static int PNEUMATICS_HUB = 42;
 	/** DIO not CAN */
```

Comment:
Add a note about what DIO is

Reply:
Thats a wiki kind of thing, not a comment thing. We can do that later.